# HG changeset patch
# User Rik <rik@octave.org>
# Date 1412102748 25200
#      Tue Sep 30 11:45:48 2014 -0700
# Node ID 8661727db27afaf1327eb2331f2a7664c4f711b4
# Parent  8a8a7bc2a09d50c9cd17bb0833c4ce1e0dad94ec
xor.m: Fix failing BIST test.

* xor.m: Fix failing BIST test.

diff --git a/scripts/miscellaneous/xor.m b/scripts/miscellaneous/xor.m
--- a/scripts/miscellaneous/xor.m
+++ b/scripts/miscellaneous/xor.m
@@ -87,15 +87,15 @@ endfunction
 
 %!assert (xor ([1, 1, 0, 0], [0, 1, 0, 1]), logical ([1, 0, 0, 1]))
 %!assert (xor ([i, i, 0, 0], [1, 0, 1, 0]), logical ([0, 1, 1, 0]))
 
 %!assert (xor (eye (2), fliplr (eye (2))), logical (ones (2)))
 %!assert (xor (speye (2), fliplr (speye (2))), sparse (logical (ones (2))))
 
 ## Test XOR reduction
-%!assert (xor ([1 0], [1 1], [0 0]), [0 1])
+%!assert (xor ([1 0], [1 1], [0 0]), logical ([0 1]))
 
 %% Test input validation
 %!error xor ()
 %!error xor (1)
 %!error <X and Y must be of compatible size> xor (ones (3,2), ones (2,3))
 
